core/src/err.rs

```diff
@@ -40,7 +40,7 @@ pub enum Error {
     QueryCursorUpdate(mongodb::error::Error),
     #[error("Getting metadata for query failed with error: {0}")]
     QueryDeserialization(mongodb::bson::de::Error),
-    #[error("Trying to execute query failed with error: {0}")]
+    #[error("Trying to execute query failed with error: {0:?}")]
```

Comment:
I noticed a lot of the displays for query errors were just "user defined error", so I changed it to use debug to hopefully get us more info when it comes up.